### DISCUSSION TITLE
Updating Vue (or any JS framework) version little by little

### DISCUSSION THREAD

**Discussion by tassioFront** [nrwl/nx#14434] [General]
Hi, Guys.
**Context**
We have a project using Vue 2.6 + Vuetify  2 + Nx 14.8.4. Our apps and libs have been created using [this community lib](https://github.com/ZachJW34/nx-plus).

**Problem**
We going to update the de Vue version (to 3), but we’d like to do that step by step.

We have thought about creating new libs using Vue 3 and using it just there. In other words, others libs and apps wouldn’t have access to Vue 3 version. I'm not sure if it's the bestest ideia. Any suggestion?

**Comment by JFGHT** [comment]
As far as I know, there's no official support for Vue yet.
These are the available packages: https://nx.dev/packages

You'd have to use unofficial libraries just like the one you mention or this one:
https://github.com/samatechtw/nx-vue3-vite/tree/main

**Comment by tino-tg** [comment]
NX now has official [Vue support](https://nx.dev/nx-api/vue). If this is still a topic for you, then I guees this is great news for you @tassioFront

**Reply by louiss0** [reply]
They decided to support Tailwind but Not unoCSS please put a thumbs up for my discussion #19888.

**Comment by tassioFront** [comment]
Yeah, @tino-tg . I'm going to test it out and see if it helps. Thanks!
FYI @MayaraRMA

**Comment by louiss0** [comment]
They decided to support Tailwind but Not unoCSS please put a thumbs up for my discussion #19888.
